Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ however, it has to be formatted properly to pass verification tests.
 
 - Fixed a problem where explicitly switching to the already active control scheme and device set for PlayerInput would cancel event callbacks for no reason when the control scheme switch would have no practical effect. This fix detects and skips device unpairing and re-pairing if the switch is detected to not be a change to scheme or devices. ([case 1342297](https://fogbugz.unity3d.com/f/cases/1342297/))
 - Any unhandled exception in `InputManager.OnUpdate` failing latter updates with `InvalidOperationException: Already have an event buffer set! Was OnUpdate() called recursively?`. Instead the system will try to handle the exception and recover into a working state.
+- Fixed a problem where only using runtimes that are not XR supported causes a compile error.This fix adds back in ENABLE_VR checks to prevent this case ([case 1368300](https://fogbugz.unity3d.com/f/cases/1368300))
```

Comment:
Remove the link as it's for private tracker, while you're at it also change one above you to public one https://issuetracker.unity3d.com/issues/input-system-button-clicks-are-missed-when-using-switchcurrentcontrolscheme-inside-of-onmousedown please